.github/actions/vpn-azure/action.yml

```diff
@@ -41,6 +41,8 @@ runs:
       shell: bash
 
     - uses: josiahsiegel/action-connect-ovpn@794339aff94452216c97f609476c367a43a31295
+    ## emvaldes@hotmail.com (Replace) - uses: ./.github/actions/action-connect-ovpn
```

Comment:
corp email please

Reply:
I have updated the email address to reflect the Aquia corporate email.